src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx

```diff
@@ -22,16 +22,14 @@ import SearchBar from 'app/components/searchBar';
 import {parseAddress, getImageRange} from 'app/components/events/interfaces/utils';
 import ImageForBar from 'app/components/events/interfaces/imageForBar';
 import {t, tct} from 'app/locale';
-import ClippedBox from 'app/components/clippedBox';
 import {IconWarning} from 'app/icons';
 import {Organization, Project, Event, Frame} from 'app/types';
 
 import DebugImage from './debugImage';
 import {getFileName} from './utils';
 
 const MIN_FILTER_LEN = 3;
-const DEFAULT_CLIP_HEIGHT = 560;
-const PANEL_MAX_HEIGHT = 600;
+const PANEL_MAX_HEIGHT = 500;
```

Comment:
should we set `400` to be the same size as the breadcrumb list? or increase the breadcrumbs list height?

Reply:
Agreed, done and done.

Reply:
thanks! 🙏